internal/services/network/virtual_network_resource.go

```diff
@@ -430,6 +467,21 @@ func flattenVirtualNetworkDDoSProtectionPlan(input *network.VirtualNetworkProper
 	}
 }
 
+func flattenVirtualNetworkEncryption(encryption *network.VirtualNetworkEncryption) interface{} {
+	if encryption == nil {
+		return nil
```

Comment:
this means we won't be outputting this correctly
```suggestion
		return make([]interface{}, 0)
```

Reply:
updated